backend/src/emissions/models.py

```diff
@@ -320,8 +318,8 @@ class Heating(models.Model):
     )
     fuel_type_choices = [(x.name, x.value) for x in HeatingFuel]
     fuel_type = models.CharField(max_length=20, choices=fuel_type_choices, blank=False)
-    unit_choices = [(x.name, x.value) for x in Unit]
-    unit = models.CharField(max_length=20, choices=unit_choices, blank=False)
+    # unit_choices = [(x.name, x.value) for x in Unit]
```

Comment:
I think we still need the unit attribute, but need to define it manually for now. Can you add the following code?

```
unit_choices = [(x, x) for x in ["kWh", "l", "kg", "m^3"]]
unit = models.CharField(max_length=20, choices=unit_choices, blank=False)
´´´

Reply:
fixed